IT.TFM/AzureDevOps/RestApi.cs

```diff
@@ -259,6 +263,8 @@ private async Task<string> CallApiAsync(string url, Method method = Method.Get,
                 return string.Empty;
             }
 
+            mutex.ReleaseMutex();
```

Comment:
@AndreasNVI if an exception is raised after the WaitOne() and this call, is the mutex automatically released or would we have an infinite wait?